plugins/ag-charts-generate-example-files/src/executors/generate/generator/examplesGenerator.ts

```diff
@@ -94,10 +94,14 @@ export const getGeneratedContents = async (params: GeneratedContentParams): Prom
     let indexHtml = await readFile(path.join(folderPath, 'index.html'));
     extractOptions ||= entryFile.includes('@ag-options-extract');
 
-    if (entryFile.includes('@ag-skip-fws') && internalFramework !== 'vanilla') {
-        entryFile = PLACEHOLDER_MAIN_TS;
-        indexHtml = `<div id="myChart"></div>`;
-        extractOptions = false;
+    if (entryFile.includes('@ag-skip-fws')) {
```

Comment:
Can we use `provided/modules` here?

Reply:
We could - however after some discussion David and I agreed that we would keep this example for the JS/TS case as an advanced use-case.

I strongly dislike provided examples, and would prefer we either wrote examples that work with the example generator OR improve the example generator over adding provided examples, which are longer-term harder to maintain consistently.

We're using `@ag-skip-fws` in a few places where we have complex examples (mostly `-test` pages) where we don't care about FW generation cases (and they are currently broken).